### ISSUE TITLE
[Adapter Cards-Schema Explorer]: Links are looks like normal text.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7871]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Pre-Requisites:
Run the Narrator using "Control+ Windows +Enter" short key in the scan mode.

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to 'Container" link present under "Containers" from the left navigation using the tab key and activate it using Enter key.
4. Navigate to the adaptive cards present under it using the Tab key.
5. Navigate to the "Cool Link"& "Emphasis Link" using narrator
6. Observe the issue here.

Actual Result:
"Cool Link"& "Emphasis Link" are looks like normal text.

Expected Result:
"Cool Link"& "Emphasis Link" should have underline and Chevron to looks like link.

Observation:
Similar issue exist on the "submit action" button down to the ["Cool Link"& "Emphasis Link"]control, as its not looking like button looks like a normal text.

User Experience:
Keyboard users and cognitive users will be impacted and confused if buttons and links are looks like normal texts and they skip to activate the links and button thinking those are all normal text.

MAS Reference:
https://www.w3.org/TR/2018/REC-WCAG21-20180605/#use-of-color

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="965" alt="9 Links and Button are looking like normal text" src="https://user-images.githubusercontent.com/104482612/190422760-05cc8b98-2680-4008-8d23-fb6f1501c2ff.png">

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA
